.ci/prepare-settings.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+set -e
+date=$(date +"%m%d%y")
+
+# Copying release-settings.xml
+mkdir -p .ci-temp
+cp ./.ci/release-settings.xml .ci-temp/
+
+# Putting values of environment variables in .ci-temp/release-settings.xml
+replace() {
+  sed -i "s/$1/$2/g" .ci-temp/release-settings.xml
+}
+
+replace SONATYPE_USER
+replace SONATYPE_PWD
+replace SOURCEFORGE_USER
+replace SOURCEFORGE_PWD
+replace GPG_PASSPHRASE
+replace GPG_KEY
+
+# Backup of settings.xml
+file1="./.ci/release-settings.xml"
```

Comment:
Why not ci-temp?

Reply:
Yes, you are right. I need to use `.ci-temp/release-settings.xml`

Changes Done